utils/installer/config.example-no-ts.lua

```diff
@@ -61,6 +61,21 @@ lvim.builtin.treesitter.highlight.enabled = true
 --   --Enable completion triggered by <c-x><c-o>
 --   buf_set_option("omnifunc", "v:lua.vim.lsp.omnifunc")
 -- end
+-- you can overwrite the null_ls setup table (useful for setting the root_dir function)
+-- lvim.lsp.null_ls.setup = {
+--   root_dir = require("lspconfig").util.root_pattern("Makefile", ".git", "node_modules"),
+-- }
+-- or if you need something more advanced
+-- lvim.lsp.null_ls.setup.root_dir = function(fname)
+--   if vim.bo.filetype == "javascript" then
+--     return require("lspconfig/util").root_pattern("Makefile", ".git", "node_modules")(fname)
+--       or require("lspconfig/util").path.dirname(fname)
+--   elseif vim.bo.filetype == "php" then
+--     return require("lspconfig/util").root_pattern("Makefile", ".git", "composer.json")(fname) or vim.fn.getcwd()
+--   else
```

Comment:
Why did you drop the [fallback](https://github.com/jose-elias-alvarez/null-ls.nvim/blob/9ffbb7db173b9c3441efa3cd6378a094460d0b7b/lua/null-ls/lspconfig.lua#L17)

Also, since else after return is not useful, this can be
`return util.root_pattern("Makefile", ".git")(fname) or util.path.dirname(fname)`

Reply:
That code does not represent my personal config. I didn't put much thought into it. I just wanted to add as much random stuff there as possible to show what you can do with it:
* different setup depending on filetype
* ability to use both `require("lspconfig/util").root_pattern` and basic stuff like `vim.fn.getcwd()`

I can replace that example code with anything you want, if you can come up with example that both makes sense and demonstrates how it might be useful.

Reply:
I think the example is great, just the else part should be what null-ls defines as default
`return util.root_pattern("Makefile", ".git")(fname) or util.path.dirname(fname)`